core/model/src/main/java/org/eclipse/rdf4j/model/util/Literals.java

```diff
@@ -580,6 +582,22 @@ public static boolean isLanguageLiteral(Literal literal) {
 		return Objects.requireNonNull(literal, "Literal cannot be null").getLanguage().isPresent();
 	}
 
+	/**
+	 * Normalises the given <a href="https://tools.ietf.org/html/bcp47">BCP47</a> language tag according to
+	 * the rules defined by the JDK in the {@link Locale} API.
+	 * 
+	 * @param languageTag
+	 *        An unnormalised, valid, language tag
+	 * @return A normalised version of the given language tag
+	 * @throws IllformedLocaleException
+	 *         If the given language tag is illformed according to the rules specified in BCP47.
+	 */
+	public static String normaliseBCP47Tag(String languageTag)
```

Comment:
I would prefer not to remove BCP47 without adding Language in its place, if that makes sense. 

On another note, I have just noticed that I used the Australian English spelling of normalise, instead of the US English that we use in the rest of the API, normalize, so I will need to change that to stay consistent.

normalizeLanguageTag?